docs/UserGuide.md

```diff
@@ -619,69 +604,53 @@ more flexibility.
 **:warning: Caution:**<br>
 
 Edit may cause **information loss**. 
-Before you type the save command, make sure there is no accidental overwrite with faulty information. 
+Before you type the save command, please check that the details of the fosterer are correct.
 
 </div>
 
 <br/>
 
-Here is a summary of how to edit a fosterer Alex Yeoh who is currently at index 1:
-
-Step 1. Enter the profile page with `view INDEX` or `edit INDEX`. e.g. `view 1` or `edit 1`.
+If you have at least one fosterer in your address book, here is an example of a profile page you will see after entering `view 1` or `edit 1`:
 
 <div style="text-align: center">
 <img src="images/screenshots/View1.png" height="480" class="center"/>
 </div>
 
-<div style="page-break-after: always;"></div>
-
-Step 2. Type in the name of the field you want to edit onto the command box. This will make the textbox visible next to the field and automatically set the text cursor to the textbox. e.g. Enter 'name'.
-
-<div style="text-align: center">
-<img src="images/screenshots/EnterName.png" height="480" class="center"/>
-</div>
+To learn more about the profile page, please refer to the section [User Interface: The Profile Page](#the-profile-page). 
 
-<br>
+<br> 
 
 <div style="page-break-after: always;"></div>
 
-Step 3. Edit the original value to a new value. e.g. From 'Alex Yeoh' to 'Bob Yeoh'.
+### Saving changes in a fosterer's details: `save`
 
-<div style="text-align: center">
-<img src="images/screenshots/ChangeNameToBobYeoh.png" height="480" class="center"/>
-</div>
+Saves changes to details of the fosterer which you have made in the profile page.
 
-<br>
+<div markdown="block" class="alert alert-danger">
 
-<div style="page-break-after: always;"></div>
+**:exclamation: Important:**<br>
 
-Step 4. Press Enter to return the text cursor back to the command box. This makes the textbox disappear.
+This command only works in the profile page, which you can navigate to by executing the `view` command.
 
-<div style="text-align: center">
-<img src="images/screenshots/BobYeoh.png" height="480" class="center"/>
 </div>
 
-<br>
+Format: `save`
 
-<div style="page-break-after: always;"></div>
+<div markdown="block" class="alert alert-info">
 
-Step 5. Enter `save` to save your changes.
+**:information_source: Notes about the command:**<br>
 
-<div style="text-align: center">
-<img src="images/screenshots/SaveBobYeoh.png" height="480" class="center"/>
+*  Entering <code>save</code> in [the profile of a new fosterer](#adding-a-fosterer-through-the-profile-page-add) saves the new fosterer and exits the profile page.
+*  Entering <code>save</code> in [the profile of an already existing fosterer](#editing-a-fosterers-details-through-the-profile-page-edit) saves the changes but does not exit the profile page, in case you want to edit more details.
```

Comment:
instead of "in case you want to edit more details" I think smth like "so that you can continue editing more details" sounds more formal, wdyt